### PR TITLE
Bf 75 alpha banner

### PR DESCRIPTION
**Pull request by shewood**
# Description

[Create banner based off Figma design](https://bf-decd.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=BF&modal=detail&selectedIssue=BF-75)

Added the alpha or Early Access banner across the top of the page. 

## Test Instructions

1. Steps on...
1. how to test...
1. or inspect this...
1. feature manually.

## Meets Definition of Done
- [x ] Checked against style guide as currently understood
- [ x] Acceptance criteria met
- [x ] Unit tests up to date
- [x ] E2E tests up to date
- [ ] Security up to standards
- [x ] I18n

### CODE REVIEW

File: components/organisms/Layout.js [hunk 1]
```diff
@@ -20,6 +21,20 @@ export const Layout = ({
   return (
     <div className="overflow-x-hidden">
       <header>
+        {/* layout for the Early Access Alpha banner start */}
+        <div className="bg-gray-100">
+          <div className="layout-container ">
+            <AlphaBanner
+              alphaBannerText={t("alphaBannerText")}
+              alphaBannerDesc={t("alphaBannerDesc")}
+              feedbackButton={`/benefit/`}
```
Comment (DaceyTom2) [inline components/organisms/Layout.js:30]: I think this was meant to bring you to the alpha page

File: cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (DaceyTom2) [inline cypress/support/commands.js:1]: Not sure this file needs to be included

File: cypress/support/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// ***********************************************************
```
Comment (DaceyTom2) [inline cypress/support/index.js:1]: Not sure this file needs to be included

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]
Looks good, one link brought me to /benefit which I believe was unintended

File: cypress/support/commands.js (hunk 2 above)
Comment (shewood) [reply cypress/support/commands.js:1]: yes, this file is needed if you want to write any custom commands. There aren't any right now though

File: cypress/support/index.js (hunk 3 above)
Comment (shewood) [reply cypress/support/index.js:1]: you need this file to add extensions like cypress-axe.

File: components/organisms/Layout.js (hunk 1 above)
Comment (shewood) [reply components/organisms/Layout.js:30]: it's just a banner

**Review by DaceyTom2 — APPROVED** [review]

**Pull request merged**